app/controllers/admin/payments_controller.rb

```diff
@@ -133,26 +133,37 @@ def create_invoice(csv, payments, payment_type, start_date, end_date = nil)
         p = Participant.where(member: payment.member, activity_id: activity_id).first
         # now create every transaction row with the following date: Blank, ledger account,
         # transaction description, VAT number, amount, cost_location ()
-        csv <<  if p.activity.group.nil? || Group.first.ledgernr.blank?
-                  ["", "1302", "#{ p.activity.name } - #{ p.member_id }", p.activity.VAT,
-                   p.currency, ""]
-                else
-                  ["", p.activity.group.ledgernr, "#{ p.activity.name } - #{ p.member_id }",
-                   p.activity.VAT, p.currency, p.activity.group.cost_location]
-                end
+        csv << if Activity.where(id: activity_id).description == "Lidmaatschap"
+                 # Dislike this way of doing it but need better ways to find membership activities
```

Comment:
inside this if-statement we know that it the `introduction` activity right?
We can set the ledger number to 8000 per default, and the vat to 0 then, removing the inner if-statement

Reply:
Yeah i was thinking the same, but the possible issue could be if we change this for whatever reason we'd have to change the code but with this we could just change the group. I'm fine with either but your solution would be probably be fine aswell so i'll change it :)

Reply:
That is a fair argument.
The problem however is that the `Lidmaatschap` activity does not have group assigned when it's created:
https://github.com/svsticky/constipated-koala/blob/fffbcac81e339e2f47e3ea4bcae827b9a647ecaa/lib/tasks/admin.rake#L55-L62
so that should still be added then.